f600233d1b154d507b782f84604bcb12/dictionary/x_g_dis_atat_igce_estimate.xml

```diff
@@ -36,23 +36,24 @@
         </element>
         <element label="Unit Price" max_length="20" name="unit_price" type="currency"/>
         <element label="Unit Quantity" max_length="4000" name="unit_quantity" type="string"/>
+        <element default="NO" label="Updated Description" max_length="1000" name="updated_description" type="string"/>
```

Comment:
This should be a of type Choice rather than String, since this is basically a dirty flag with Yes/No options